### ISSUE TITLE
Vertices Not Indexed Properly?

### ISSUE THREAD

**Issue by maxattack** [nem0/OpenFBX#57]
Some vertices appear indexed improperly  (a handful out of ~1k, so it's <1%?)

No triangulation going on. However, even so, the indexing code is hard to follow as I debug this issue. Is there any guidance you can give me for possible leads?

(Left: OpenFBX imported geometry, Right: same FBX imported into maya)
![image](https://user-images.githubusercontent.com/798685/85340870-c5390380-b49b-11ea-8567-17e5b2262084.png)

**Comment by maxattack** [comment]
No malformed-data warnings importing into maya or anything.

**Comment by nem0** [comment]
Without triangulation indices/vertices are basically raw data from fbx, so it's probably something else. The code for that was contributed by some other user, so I don't guarantee it's correct. If you want you can upload the model and I will take a look. 
Note: raw indices have negative values marking end of polygon: https://github.com/nem0/OpenFBX/blob/master/src/ofbx.cpp#L2299

**Comment by maxattack** [comment]
Yeah, I'm accounting for those.  BTW, `(-idx - 1)`  is just bitwise complement, so you can use `~idx` and save some instructions.


[mecha.zip](https://github.com/nem0/OpenFBX/files/4816112/mecha.zip)

**Comment by nem0** [comment]
Note: singed integer binary representation before c++20 is implementation defined

**Comment by nem0** [comment]
I tried to import the model in Lumix Engine and it seems ok (although it's with triangulation)
